crates/matrix-sdk-ui/src/timeline/tests/edit.rs

```diff
@@ -165,13 +166,15 @@ async fn test_edit_updates_encryption_info() {
     let timeline = TestTimeline::new();
     let event_factory = &timeline.factory;
 
+    let room_id = room_id!("!room:id");
     let original_event_id = event_id!("$original_event");
 
-    let mut original_event = event_factory
+    let original_event = event_factory
         .text_msg("**original** message")
         .sender(*ALICE)
         .event_id(original_event_id)
-        .into_sync();
+        .room(room_id)
+        .into_raw_timeline();
```

Comment:
since we can no longer poke values directly into `original_event.encryption_info` below, the easiest way to do what this test wants is to build an `AnyTimelineEvent` instead of an `AnySyncTimelineEvent`, wrap that into a `DecryptedRoomEvent`, and then use `.into()` to turn *that* into a `SyncTimelineEvent`.